src/features/ci-cd/hooks/track-latest-release.ts

```diff
@@ -107,24 +116,23 @@ const updateLatestRelease = async (
 hooks.addPureHook('PATCH', 'resin', 'release', {
 	POSTRUN: async (args) => {
 		const { request } = args;
-		// If we're updating a build by id and setting it successful then we update the application to this build
+		// If we're updating a release by id and setting it successful then we update the application to this build
 		if (request.odataQuery != null) {
 			const keyBind = request.odataQuery.key;
 			// TODO: Support named keys
 			if (keyBind != null && 'bind' in keyBind) {
```

Comment:
I think you can ditch both of these ifs since they were only as a way to get the id and we don't care about that anymore, instead you can add a `request.affectedIds.length > 0` check just to make sure we did at least update something